core/lib/config/src/configs/contracts.rs

```diff
@@ -37,6 +37,7 @@ pub struct ContractsConfig {
     pub l2_testnet_paymaster_addr: Option<Address>,
     pub l1_multicall3_addr: Address,
     pub ecosystem_contracts: Option<EcosystemContracts>,
+    // Used for the RPC API and by the BaseTokenAdjuster. If not set, ETH is the chain's base token.
```

Comment:
for ETH based chain, it is not None but defaulted to “0x0..01”

Reply:
thanks! changed :)